src/Core/Command/Repo/SetOptionsCommand.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace DigipolisGent\Github\Core\Command\Repo;
+
+use DigipolisGent\Github\Core\Handler\RepositoriesHandler;
+use DigipolisGent\Github\Core\Log\ConsoleLogger;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Set some options for the repositories of a team.
+ *
+ * @package DigipolisGent\Github\Command\Repo
+ */
+class SetOptionsCommand extends ListCommand
```

Comment:
I think it would be better to extend AbstractCommand, and maybe move getRepositories to AbstractCommand, since that will most likely be needed in almost all cases.
@zero2one